internal/agent/install.go

```diff
@@ -331,3 +339,14 @@ func prepareConfiguration(ctx context.Context, source string) (string, error) {
 
 	return f.Name(), nil
 }
+
+func generateInstallConfForCLIArgs(source string) string {
+	if source == "" {
+		return ""
+	}
+
+	return fmt.Sprintf(`install:
```

Comment:
maybe it's worth validating if source is a real uri?

Reply:
I see a validation at the gathering of the source, so this comment can be ignored

Reply:
This part of the code makes a config out of `--source` cli argument. If its' invalid, they user will get whatever error they would get if they put it in the Kairos config themselves. As such, this function's responsibility is only to make it part of the configuration and not validate it (which should happen later).